### PR TITLE
Deprecate AC documentation

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2028

I moved all dedicated AC docs to the documentation archive and set up redirects. The only AC-related doc now on our site is "Migrate to Astro Runtime"

### CODE REVIEW

File: archived_software_docs/astronomer-certified/ac-cve.md [hunk 1]
```diff
@@ -5,11 +5,8 @@ id: ac-cve
 description: "Common Vulnerabilities and Exposures identified within our Astronomer Certified images."
 ---
 
-:::caution
-
-Astronomer will no longer release new versions of AC starting with the release of Apache Airflow 2.4. Astronomer recommends creating all new Deployments with Astro Runtime, as well as migrating existing Deployments from AC to Astro Runtime as soon as your organization is ready. See [Migrate to Runtime](migrate-to-runtime.md) and [Runtime image architecture](runtime-image-architecture.md).
-
-:::
+> **Warning**
+> No versions of Astronomer Certified (AC) are currently supported by Astronomer. Astronomer stopped releasing new versions of AC with the release of Apache Airflow 2.4. Astronomer recommends creating all new Deployments with Astro Runtime, as well as migrating existing Deployments from AC to Astro Runtime as soon as your organization is ready. See [Migrate to Runtime](migrate-to-runtime.md) and [Runtime image architecture](runtime-image-architecture.md).
```
Comment (paolaperaza) [inline archived_software_docs/astronomer-certified/ac-cve.md:9]:
```suggestion
::: warning

No versions of Astronomer Certified (AC) are currently supported by Astronomer. Astronomer stopped releasing new versions of AC with the release of Apache Airflow 2.4. Astronomer recommends that you create all new Deployments with Astro Runtime and migrate existing Deployments from AC to Astro Runtime as soon as your organization is ready. See [Migrate to Runtime](migrate-to-runtime.md) and [Runtime image architecture](runtime-image-architecture.md).
```

Format?

File: archived_software_docs/astronomer-certified/ac-support-policy.md [hunk 2]
```diff
@@ -5,11 +5,8 @@ id: ac-support-policy
 description: Versioning and lifecycle policy for Astronomer Certified, our Apache Airflow offering for Astronomer Software.
 ---
 
-:::caution
-
-Astronomer will no longer release new versions of AC starting with the release of Apache Airflow 2.4. Astronomer recommends creating all new Deployments with Astro Runtime, as well as migrating existing Deployments from AC to Astro Runtime as soon as your organization is ready. See [Migrate to Runtime](migrate-to-runtime.md) and [Runtime image architecture](runtime-image-architecture.md).
-
-:::
+> **Warning**
+> No versions of Astronomer Certified (AC) are currently supported by Astronomer. Astronomer stopped releasing new versions of AC with the release of Apache Airflow 2.4. Astronomer recommends creating all new Deployments with Astro Runtime, as well as migrating existing Deployments from AC to Astro Runtime as soon as your organization is ready. See [Migrate to Runtime](migrate-to-runtime.md) and [Runtime image architecture](runtime-image-architecture.md).
```
Comment (paolaperaza) [inline archived_software_docs/astronomer-certified/ac-support-policy.md:9]: Same here

File: software/manage-airflow-versions.md [hunk 3]
```diff
@@ -67,18 +67,15 @@ The Software UI and CLI only provide Airflow versions that are later than the ve
 
     <pre><code parentName="pre">{`FROM quay.io/astronomer/astro-runtime:${siteVariables.runtimeVersion}`}</code></pre>
 
-    For a list of currently supported Astronomer images, see:
-
-    - [Astronomer Certified lifecycle schedule](ac-support-policy.md#astronomer-certified-lifecycle-schedule)
-    - [Astro Runtime lifecycle schedule](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule)
+    For a list of currently supported Astronomer images, see [Astro Runtime maintenance and lifecycle policy](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule).
```
Comment (paolaperaza) [inline software/manage-airflow-versions.md:70]:
```suggestion
    For a list of supported Astro Runtime versions, see [Astro Runtime maintenance and lifecycle policy](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule).
```

File: software/manage-airflow-versions.md [hunk 4]
```diff
@@ -67,18 +67,15 @@ The Software UI and CLI only provide Airflow versions that are later than the ve
...
 
   :::warning
 
   After you upgrade your Airflow version, you can't revert to an earlier version.
 
   :::
 
-3. Optional. Test your upgrade on your local machine by running:
+1. Optional. Test your upgrade on your local machine by running:
```
Comment (paolaperaza) [inline software/manage-airflow-versions.md:78]: Double check steps here

File: software/migrate-to-runtime.md [hunk 5]
```diff
@@ -7,10 +7,12 @@ description: Run an upgrade progress to migrate your Software Deployment from As
 
 import {siteVariables} from '@site/src/versions';
 
-Astro Runtime builds on the reliability of Astronomer Certified (AC) with new features that center on usability and performance.
+All versions of Astronomer Certified (AC) are no longer supported. Astronomer recommends migrating all of your Deployments to use an Astro Runtime image as soon as possible. Astro Runtime builds on the reliability of AC with new features that center on usability and performance.
```
Comment (paolaperaza) [inline software/migrate-to-runtime.md:10]:
```suggestion
All versions of Astronomer Certified (AC) are no longer supported on Astronomer Software. Astronomer recommends migrating all of your Deployments to use an Astro Runtime image as soon as possible. Astro Runtime builds on the reliability of AC with new features that center on usability and performance. 
```

File: software/migrate-to-runtime.md [hunk 6]
```diff
@@ -57,18 +59,15 @@ If you prefer to use the Astro CLI, you can run `astro deployment runtime migrat
     FROM quay.io/astronomer/astro-runtime:5.0.4
     ```
 
-    For a list of currently supported Astronomer images, see:
-
-    - [AC Lifecycle Schedule](ac-support-policy.md#astronomer-certified-lifecycle-schedule)
-    - [Astro Runtime Lifecycle Schedule](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule)
+    For a list of currently supported Astronomer images, see [Astro Runtime maintenance and lifecycle policy](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule).
```
Comment (paolaperaza) [inline software/migrate-to-runtime.md:62]:
```suggestion
    For a list of supported Astro Runtime versions, see [Astro Runtime maintenance and lifecycle policy](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule).
```

File: software/release-lifecycle-policy.md [hunk 7]
```diff
@@ -17,7 +17,7 @@ For information on the latest Astronomer Software releases, see [Release notes](
 
 :::info
 
-These policies apply only to the Astronomer Software platform. For release and lifecycle policies related to Astronomer images, see [Astronomer Certified versioning and support](ac-support-policy.md) and [Runtime release and lifecycle Policy](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy).
+These policies apply only to the Astronomer Software platform. For release and lifecycle policies related to Astronomer images, see [Runtime release and lifecycle Policy](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy).
```
Comment (paolaperaza) [inline software/release-lifecycle-policy.md:20]:
```suggestion
These policies apply only to the Astronomer Software platform. For release and lifecycle policies related to Astro Runtime, see [Runtime release and lifecycle Policy](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy).
```

File: software_versioned_docs/version-0.30/manage-airflow-versions.md [hunk 8]
```diff
@@ -67,18 +67,15 @@ The Software UI and CLI only provide Airflow versions that are later than the ve
 
     <pre><code parentName="pre">{`FROM quay.io/astronomer/astro-runtime:${siteVariables.runtimeVersion}`}</code></pre>
 
-    For a list of currently supported Astronomer images, see:
-
-    - [Astronomer Certified lifecycle schedule](ac-support-policy.md#astronomer-certified-lifecycle-schedule)
-    - [Astro Runtime lifecycle schedule](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule)
+    For a list of currently supported Astronomer images, see [Astro Runtime lifecycle schedule](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule).
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.30/manage-airflow-versions.md:70]:
```suggestion
    For a list of currently supported versions of Astro Runtime, see [Astro Runtime lifecycle schedule](/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule).
```

File: software_versioned_docs/version-0.30/release-lifecycle-policy.md [hunk 9]
```diff
@@ -17,7 +17,7 @@ For information on the latest Astronomer Software releases, see [Release notes](
 
 :::info
 
-These policies apply only to the Astronomer Software platform. For release and lifecycle policies related to Astronomer images, see [Astronomer Certified versioning and support](ac-support-policy.md) and [Runtime release and lifecycle Policy](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy).
+These policies apply only to the Astronomer Software platform. For release and lifecycle policies related to Astronomer images, see and [Runtime release and lifecycle Policy](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy).
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.30/release-lifecycle-policy.md:20]:
```suggestion
These policies apply only to the Astronomer Software platform. For release and lifecycle policies related to Astro Runtime, see and [Runtime release and lifecycle Policy](astro/runtime-version-lifecycle-policy).
```

File: software_versioned_docs/version-0.30/migrate-to-runtime.md [hunk 10]
```diff
@@ -55,18 +55,15 @@ If you prefer to use the Astro CLI, you can run `astro deployment runtime migrat
     FROM quay.io/astronomer/astro-runtime:5.0.4
     ```
 
-    For a list of currently supported Astronomer images, see:
-
-    - [AC Lifecycle Schedule](ac-support-policy.md#astronomer-certified-lifecycle-schedule)
-    - [Astro Runtime Lifecycle Schedule](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule)
+    For a list of currently supported Astronomer images, see [Astro Runtime Lifecycle Schedule](https://docs.astronomer.io/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule).
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.30/migrate-to-runtime.md:58]:
```suggestion
    For a list of currently supported versions of Astro Runtime, see [Astro Runtime Lifecycle Schedule](/astro/runtime-version-lifecycle-policy#astro-runtime-lifecycle-schedule).
```

File: software_versioned_docs/version-0.30/version-compatibility-reference.md [hunk 11]
```diff
@@ -29,10 +29,7 @@ While the tables below reference the minimum compatible versions, we typically r
 
 For more detail about the changes in each Astronomer Software release, see the [Astronomer Software Release Notes](release-notes.md).
 
-All currently supported Astronomer-distributed images are compatible with all versions of Astronomer Software. Astronomer Certified and Astro Runtime maintenance is independent of Software maintenance. For more information, see:
-
-- [Astro Runtime maintenance and lifecycle policy](runtime-version-lifecycle-policy.md)
-- [Astronomer Certified versioning and support](ac-support-policy.md)
+All currently supported Astronomer-distributed images are compatible with all versions of Astronomer Software. Astronomer Certified and Astro Runtime maintenance is independent of Software maintenance. For more information, see [Astro Runtime maintenance and lifecycle policy](runtime-version-lifecycle-policy.md).
```
Comment (paolaperaza) [inline software_versioned_docs/version-0.30/version-compatibility-reference.md:32]:
```suggestion
All currently supported versions of Astronomer-distributed images are compatible with all versions of Astronomer Software. Astronomer Certified and Astro Runtime maintenance is independent of Software maintenance. For more information, see [Astro Runtime maintenance and lifecycle policy](runtime-version-lifecycle-policy.md).
```

**Review by paolaperaza — APPROVED** [review]

File: software/migrate-to-runtime.md (hunk 5 above)
Comment (jwitz) [reply software/migrate-to-runtime.md:10]: It's across all platforms though right? Trying to catch folks using the PyPI distribution as well.

**Review by paolaperaza — APPROVED** [review]

**Review by lzdanski — APPROVED** [review]
Did a final editorial pass - LGTM

**Pull request merged**